CHANGELOG.md

```diff
@@ -1,23 +1,21 @@
-## [CAPI Control Plane Provider Talos 0.4.0-beta.0](https://github.com/talos-systems/cluster-api-control-plane-provider-talos/releases/tag/v0.4.0-beta.0) (2021-11-29)
+## [CAPI Control Plane Provider Talos 0.4.0-beta.1](https://github.com/talos-systems/cluster-api-control-plane-provider-talos/releases/tag/v0.4.0-beta.1) (2021-12-09)
 
-Welcome to the v0.4.0-beta.0 release of CAPI Control Plane Provider Talos!  
+Welcome to the v0.4.0-beta.1 release of CAPI Control Plane Provider Talos!  
 *This is a pre-release of CAPI Control Plane Provider Talos*
 
 
 
 Please try out the release binaries and report any issues at
 https://github.com/talos-systems/cluster-api-control-plane-provider-talos/issues.
 
-### `init` nodes deprecation
+### CAPI v1beta1
 
-Starting from this release CACPPT no longer relies on init nodes to bootstrap the cluster.
-Instead, it is now issues bootstrap and keeps bootstrap status information in the `TalosControlPlane` resource.
-Bootstrap state can also be retrieved from the conditions.
+This release of CACPPT brings compatibility with CAPI v1beta1.
 
 
-### CAPI v1beta1
+### Init Node Deprecation
 
-This release of CACPPT brings compatibility with CAPI v1beta1.
+This release of CACPPT deprecates the `init` node that was used in previous Talos Linux designs. Specifying only a `controlplane` as part of the spec is now all that's required and the provider itself will handle automatic bootstrap of the first controlplane node.
```

Comment:
I guess we can also mention which version of talos started deprecating `init` config